packages/client/src/sync/fetcher/accountfetcher.ts

```diff
@@ -287,6 +290,11 @@ export class AccountFetcher extends Fetcher<JobTask, AccountData[], AccountData>
     const origin = this.getOrigin(job)
     const limit = this.getLimit(job)
 
+    if (this.highestKnownHash && compareBytes(limit, this.highestKnownHash) < 0) {
```

Comment:
ahh so this solves the problem that if some range is empty it won't be requested :+1: